civihr_default_theme/assets/sass/partials/_navs.scss

```diff
@@ -0,0 +1,25 @@
+// Styles for Navs
+// --------------------------------------------------
+
+.nav-tabs-header {
+    box-shadow: 0px 3px 10px 0px rgba(0,0,0,0.035),0px 4px 8px 0px rgba(0,0,0,0.035);
```

Comment:
Please omit the units when the value is 0: `0px` -> `0`, and when you have multiple values leave a space between one value and the other:
```sass
box-shadow: {first value}, {second value}
```